resources/aws/internet_gateway.go

```diff
@@ -1,11 +1,13 @@
 package aws
 
 import (
+	"context"
 	"fmt"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
-	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/backoff"
```

Comment:
Wow this is ancient! Good to sort it so we can move forward.

It's going to be a happy day with v1 gets deleted! 😄